### PR TITLE
add titatoggle v1.2.9 with git autoupdate config

### LINKED ISSUE #7859: [Request] Add titatoggle
**Library name:** titatoggle
**Git repository url:** https://github.com/kleinejan/titatoggle
**npm package url(optional):** 
**License(s):** BSD-2-Clause
**Official homepage:** http://kleinejan.github.io/titatoggle/
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7859 
@kennynaoh Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/kleinejan/titatoggle
 * Official website (optional, not the repository): https://kleinejan.github.io/titatoggle/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 62
   - Count of watchers: 5
   - Count of forks: 29
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7859, cc @kleinejan

### CODE REVIEW

**Comment by kennynaoh** [comment]
@PeterDaveHello 
I think this PR is correct.
Please help me review it again.
Thanks.

File: ajax/libs/titatoggle/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "buttons",
+    "toggle"
+  ],
+  "license": "BSD-2-Clause",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kleinejan/titatoggle.git",
+    "basePath": "dist",
+    "files": [
+      "**/*.!(less)"
```
Comment (PeterDaveHello) [inline ajax/libs/titatoggle/package.json]: Just asking, have you really run the autoupdate to test this config? Thanks.

File: ajax/libs/titatoggle/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "css",
+    "less",
+    "bootstrap",
+    "togglebutton",
+    "checkbox",
+    "form",
+    "buttons",
+    "toggle"
+  ],
+  "license": "BSD-2-Clause",
```
Comment (PeterDaveHello) [inline ajax/libs/titatoggle/package.json]: https://github.com/kleinejan/titatoggle/commit/5245266c46ca6a8a7d32361ae6ade8845929e0b7

File: ajax/libs/titatoggle/package.json (hunk 1 above)
Comment (pvnr0082t) [inline ajax/libs/titatoggle/package.json]: Yes, I grab files by running auto-update.

**Comment by PeterDaveHello** [comment]
I don't think we need to wait for the response, obviously the author forgot to update license in package.json

**Comment by kleinejan** [comment]
Hi all,
Right now I'm at the next web conference so it's hard for me to answer
correctly. I  changed the licence a while ago and indeed forgot to update
package.json

**Comment by PeterDaveHello** [comment]
@pvnr0082t so just update the license.

**Comment by kleinejan** [comment]
It's updated v1.2.9

**Comment by pvnr0082t** [comment]
@kennynaoh I have updated license and lib to the latest version. Please help me review again, thank you.

**Comment by kennynaoh** [comment]
@PeterDaveHello 
I think this PR is correct.
Please help me review it again.
Thanks.

**Pull request merged**